docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1105,10 +1255,45 @@ These methods are deprecated:
 #### Implementation of ownership API
 
 In order to support the recursive publishing of dataobjects, a new API has been developed to allow
-developers to declare dependencies between objects. See the
-[versioned documentation](/developer_guides/model/versioning) for more information.
+developers to declare dependencies between objects. This is done to ensure that the published state
+of linked components are consistent with their "owner." Without the concept of ownership, these linked
+components could be implicitly exposed on the frontend, which may not align with the intent of the
+content author.
 
-By default all versioned dataobjects will automatically publish objects that they own.
+For instance, on a products page which has a list of products, the products should not be published or
+ unpublished unless the products page is, too. The ownership API solves this by allowing you to declare
+ a two-way relationship between objects, typically, but not necessarily, linked by a database relationship
+ (`has_many`, `many_many`, etc.).
+
+```php
+class ProductPage extends Page {
+    private static $has_many = [
+        'Products' => Product::class
+    ];
+    
+    private static $owns = [
+        'Products'
+    ];
+}
+
+class Product extends DataObject {
+    private static $extensions = [
+        Versioned::class
+    ];
+    
+    private static $has_one = [
+        'Parent' => ProductPage::class
+    ];
+}
+```
+If your objects are linked by something other than a database relationship, for instance, a custom
+getter that is computed at runtime, the same rules can be applied, as long as you provide an `$owned_by`
+setting on the child object.
+
+For more information, see:
```

Comment:
Please be careful with excessive whitespace (like two-element lists) - I'd suggest that's inlined. Too hard to skim read the guide otherwise